scenario/BulkerScenario.ts

```diff
@@ -7,7 +7,7 @@ import { exp } from '../test/helpers';
 scenario(
   'Comet#bulker > (non-WETH base) all non-reward actions in one txn',
   {
-    filter: async (ctx) => await isBulkerSupported(ctx) && !matchesDeployment(ctx, [{deployment: 'weth'}, {network: 'mumbai'}]),
+    filter: async (ctx) => await isBulkerSupported(ctx) && !matchesDeployment(ctx, [{deployment: 'weth'}, {network: 'mumbai'}, { network: 'linea-goerli' }]),
```

Comment:
Curious why we need to exclude Linea from this test?

Reply:
Otherwise the scenario was failing, I can't remember should I remove it ?

Reply:
Yeah I think it's better to let the scenario fail so we can see if it's an actual problem or not.